Stack 2 task output doesn't render links for dependency builds

Stack 2 task output looks something like this when building intra-project dependencies -

```
foo> [10 of 10] Compiling Foo
foo> 
foo> /path/to/Foo.hs:18:50: error:
foo>     • Couldn't match type ....

```

However, HaskForce doesn't properly detect and render the paths as links.

--- a/src/com/haskforce/run/stack/task/StackTaskCommandLineState.scala
+++ b/src/com/haskforce/run/stack/task/StackTaskCommandLineState.scala
@@ -31,7 +31,7 @@ class StackTaskCommandLineState(
         config.getProject.getBasePath,
         new RelativeDiscoveryFileHyperlinkRawDataFinder(getProject, Array(
           new PatternHyperlinkFormat(
-            Pattern.compile("^\\s*([^:]+):(\\d+):(\\d+):"), false, false,
+            StackTaskCommandLineState.LINK_TO_SOURCE_REGEX, false, false,
             PatternHyperlinkPart.PATH, PatternHyperlinkPart.LINE, PatternHyperlinkPart.COLUMN
           )
         ))
@@ -90,6 +90,11 @@ class StackTaskCommandLineState(
       })
       .toMap.asJava
   }
+}
+
+object StackTaskCommandLineState {
+  // [\\w\\-]*>? detects a dependency building; issue #409
+  private val LINK_TO_SOURCE_REGEX = Pattern.compile("^[\\w\\-\\s]*>?\\s*([^:]+):(\\d+):(\\d+):")
 }
 
 /**

